_includes/configuration/images-upload-handler.md

```diff
@@ -6,7 +6,8 @@ The upload handler function takes four arguments:
 
 * `blobInfo`
 * A `success` callback
-* A `failure` callback
+* A `failure` callback that takes an error-message and an optional object containing:
```

Comment:
Not sure if "error-message" needs the hyphen - up to @tylerkelly13

Reply:
```suggestion
* A `failure` callback that takes an error message and an optional object containing:
```